prisma-fmt/tests/text_document_completion/scenarios/extended_indexes_types_mysql/result.json

```diff
@@ -0,0 +1,10 @@
+{
+  "isIncomplete": false,
+  "items": [
+    {
+      "label": "BTree",
+      "kind": 13,
+      "detail": "Can handle equality and range queries on data that can be sorted into some ordering (default)."
```

Comment:
Do we support the `type` argument on MySQL? That's a bit weird to me.

Reply:
Yes, but it's removed during formatting round because it has the default value `BTree`...